verification/tests/sourcify.rs

```diff
@@ -34,9 +34,16 @@ async fn should_return_200() {
 
     assert!(
         resp.status().is_success(),
-        "Failed to verify contract, status is {}",
+        "failed to verify contract, status is {}",
         resp.status()
     );
 
+    let body: VerificationResponse = test::read_body_json(resp).await;
+    assert_eq!(
+        body.status,
+        VerificationStatus::Ok,
+        "invalid verification status",
+    )
```

Comment:
Is there any special reason we do not check `VerificationResult` of the response?

Reply:
because I return empty response for now:

```VerificationResult {
                contract_name: "".into(),
                compiler_version: "".into(),
                evm_version: "".into(),
                constructor_arguments: None,
                contract_libraries: None,
                abi: "".into(),
                sources: HashMap::new(),
            };```

i will change this test in next pr